bin/lp-builder.js

```diff
@@ -22,12 +22,13 @@ var config = {
   LP_VERSION: null,
   LP_APPS: null,
   LP_TASKS: ['copy', 'appmanifests', 'optimize'],
+  LP_GLOBAL_TASKS: ['copySpeechData'],
```

Comment:
Do we need global tasks in addition to regular ones?  It seems that `copySpeechData` ends up in `tasks` anyways.